### PR TITLE
Tidy up after reading contributing docs

### PR DESCRIPTION
**Pull request by nameloCmaS**
This pull request does the below:

1. Tidies the contribute docs generally (updated some out of date links, added some more detail to code examples, updated RST syntax to use code-block in place of highlight descriptor, corrected tox test environments).
2. Changed Firefox example path from canary to firefox_nightly.
3. Removed old rst file referencing install of PyQt4 for now unused driver.
4.  Removed link to non-existent rst in old releases.
5. Updated Selenium keys as example code does not work as mentioned in issue #1147.
6. Corrected tutorial to ref second btnK named element as didn't work.
7. Added missing lint environment to tox.ini

For item 7 I have rewritten the tox.ini file but will raise an issue for bigger changes (#1256).

### CODE REVIEW

File: splinter/driver/webdriver/__init__.py [hunk 1]
```diff
@@ -97,21 +97,17 @@ def next(self):  # NOQA: A003
         next_handle = self._browser.driver.window_handles[next_index]
         return Window(self._browser, next_handle)
 
-    def is_current():
```
Comment (jsfehler) [inline splinter/driver/webdriver/__init__.py:100]: Using decorators here is good, but I'd prefer if you split them into a separate PR. They're not related to the rest of the changes.

File: docs/contribute/contribute.rst [hunk 2]
```diff
@@ -103,11 +109,13 @@ Build
 The `build_docs` environment is a wrapper around Sphinx's Makefile.
 Arguments will be passed to the Makefile. Thus, to build the docs in HTML format:
 
-.. highlight:: bash
+.. code-block:: bash
+
+  tox -e build_docs -- html
```
Comment (jsfehler) [inline docs/contribute/contribute.rst:114]: Thanks for catching this, typo on my part.

**Review by jsfehler — CHANGES_REQUESTED** [review]

File: splinter/driver/webdriver/__init__.py (hunk 1 above)
Comment (nameloCmaS) [reply splinter/driver/webdriver/__init__.py]: Reverted file; will submit separately.

**Review by nameloCmaS — COMMENTED** [review]
Removed change to code as requested.

**Review by jsfehler — APPROVED** [review]
